cmd/starbridge/model/chain.go

```diff
@@ -61,7 +61,8 @@ func nextStellarNonceFn(sourceAccount string) (uint64, error) {
 	if e != nil {
```

Comment:
💭 I think the sequence number should be stored rather than pulled from Horizon. I'm a little worried we're relying on submission to have taken place for this seq num to be correct, since it will only be updated on Horizon if the last one was submitted.

I was thinking we could specify a starting sequence number as a parameter on boot and the service could work out, somehow, what sequence number it should be up to based on the next message it sees. Is there anything in the Ethereum message/event that we could use for that?

Reply:
yeah, that makes sense. we can use the "total ordering of events" on Ethereum for our seq num ordering.

now that we are expecting the user's wallet to submit the tx, what if they never submit it, do we need to have different source accounts? we should discuss how to approach this tomorrow